contracts/Ecliptic.sol

```diff
@@ -485,10 +501,22 @@ contract Ecliptic is EclipticBase, SupportsInterfaceWithLookup, ERC721Metadata
         emit Transfer(old, _target, uint256(_point));
       }
 
+      //  if we're depositing, clear L1 data so that no proxies can be used
```

Comment:
Three reasons:
- Avoid extra on-chain data usage to be a good citizen
- Reduce gas cost of depositing, since setting storage values to 0 provides a discount
- Semantically, we do this so that we don't have to check every time we use a proxy that the owner isn't L2.

Reply:
Realized you asked about the non-proxy data, and yes those are just for the first two reasons.  Claims is maybe arguable, but there's no more claims on L2 for now, so it seems fine.